cmd/get.go

```diff
@@ -122,13 +122,16 @@ func (o *GetOptions) Run() (err error) {
 		}
 	}
 
+	// convert Page and PageSize to Limit and Offset
+	listParams := lib.NewListParams("", o.Page, o.PageSize)
```

Comment:
I don't think it makes sense to construct this just to break it back apart a few lines later. Just use o.Page and o.PageSize to assign to Offset and Limit below.

Reply:
Spoke about this offline, didn't realize that o.Page and o.PageSize aren't the same values as Limit and Offset, oops!